basic/src/test/java/org/jboss/hal/testsuite/test/configuration/remoting/endpointconfiguration/EndpointConfigurationAttributesTestCase.java

```diff
@@ -160,11 +161,14 @@ public void editTransmitWindowSizeInvalidValue() throws Exception {
     }
 
     @Test
-    public void editWorker() throws Exception {
+    public void editWorkerShowsAlert() throws Exception {
         final String value = "my_worker_" + RandomStringUtils.randomAlphanumeric(7);
         final ModelNodeResult originalModelNodeResult = operations.readAttribute(ENDPOINT_CONFIGURATION_ADDRESS, WORKER);
         try {
-            enterTextAndVerify(WORKER, value);
+            ConfigFragment configFragment = page.getConfigFragment();
+            configFragment.editTextAndSave(WORKER, value);
+            AlertFragment alert = page.getAlertArea();
+            Assert.assertNotNull(alert);
```

Comment:
For the next time it would be IMHO better to extend `ConfigChecker.Builder`. But this is OK, since you are new to this...